user_docs/en/userGuide.t2t

```diff
@@ -2785,10 +2785,19 @@ You can browse installed incompatible add-ons using the [incompatible add-ons ta
 

```

Comment:
This paragraph is very clear and well phrased. Thanks @Adriani90 and @seanbudd. Do not modify anymore!